### DISCUSSION TITLE
New Search doesn't show amount of entries

### DISCUSSION THREAD

**Discussion by vlad-velichko** [josdejong/svelte-jsoneditor#360] [Ideas]
We lost wonderful functionality from old version:
![image](https://user-images.githubusercontent.com/44846794/163019403-5a77c182-f004-4911-858b-94954e743408.png)

**Comment by josdejong** [comment]
Thanks for reporting, I hadn't realized that yet. The count is indeed really useful.

I've replied to an issue about this in the CodeMirror editor: https://discuss.codemirror.net/t/get-number-of-hits-of-searchquery/3939/3. If it will not be built in in CodeMirror itself we'll have to see whether we can extend it ourselves.

**Comment by josdejong** [comment]
Help implementing the number of search results will be welcome. Anyone interested in picking this up?

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it. Help implementing this would be very welcome.
